libeventheader-tracepoint/include/eventheader/TraceLoggingProvider.h

```diff
@@ -313,8 +317,8 @@ enabled.
 */
 #define TraceLoggingProviderEnabled(providerSymbol, eventLevel, eventKeyword)  ({ \
     enum { \
-        _tlgKeywordVal = (uint64_t)(eventKeyword), \
-        _tlgLevelVal = (uint64_t)(eventLevel) \
+        _tlgKeywordEnum = (uint64_t)(eventKeyword), \
+        _tlgLevelEnum = (uint64_t)(eventLevel) \
```

Comment:
Note: There was a naming conflict between these enum values and a macro. It didn't cause any problem but it was confusing so fix it.